example/package.json

```diff
@@ -15,9 +15,8 @@
   "author": "Nick Dodson",
   "license": "MIT",
   "dependencies": {
+    "ethdeploy": "file:../",
     "ethdeploy-solc-loader": "file:../src/loaders/solc",
     "ethdeploy-environment-loader": "file:../src/loaders/environment",
-    "ethdeploy": "file:../",
-    "ethereumjs-testrpc": "*"
```

Comment:
I was intending to move it to devDependencies. Will revert the change, since Dependencies is good enough.